src/store/ChartAdapterStore.ts

```diff
@@ -537,7 +519,48 @@ export default class ChartAdapterStore {
 
     toggleXScrollBlock = (isBlocked = true) => {
         this.isXScrollBlocked = isBlocked;
-        window.flutterChart?.app.toggleXScrollBlock(isBlocked);
+        const flutterChart = document.getElementsByClassName('flutter-chart')[0] as HTMLElement;
+
+        if (flutterChart) {
+            if (isBlocked) {
+                flutterChart.style.overflowY = 'scroll';
+                flutterChart.style.overflowX = 'hidden';
+                flutterChart.style.overscrollBehavior = 'contain';
+                flutterChart.style.touchAction = 'pan-y';
+            } else {
+                this.allowScroll(flutterChart);
+            }
+        }
+    };
+
+    allowScroll = (element: HTMLElement) => {
+        let lastScrollTop = element.scrollTop;
+        let isScrolling = false;
+        let scrollTimeout: ReturnType<typeof setTimeout>;
+
+        const monitorScroll = () => {
```

Comment:
Sorry for stupid question, but don't we need to call a function? `monitorScroll` is just declared, not called

Reply:
Yeah , accidentally removed that